policy.json

```diff
@@ -0,0 +1,43 @@
+{
+  "instance_min_count": 12,
+  "instance_max_count": 50,
+  "scaling_rules": [
+    {
+      "metric_type": "throughput",
+      "breach_duration_secs": 60,
+      "threshold": 90,
+      "operator": "<",
+      "cool_down_secs": 60,
+      "adjustment": "-1"
+    },
+    {
+      "metric_type": "throughput",
+      "breach_duration_secs": 60,
+      "threshold": 90,
+      "operator": ">=",
+      "cool_down_secs": 60,
+      "adjustment": "+1"
+    }
+  ],
+  "schedules": {
+    "timezone": "Europe/London",
+    "recurring_schedule": [
+      {
+        "start_time": "04:00",
+        "end_time": "00:00",
+        "days_of_week": [1, 2, 3, 4, 5, 6, 7],
+        "instance_min_count": 12,
+        "instance_max_count": 50,
+        "initial_min_instance_count": 15
+      },
+      {
```

Comment:
Do we need this if our default at instance_min and instance_max are set out at the top? (https://github.com/cloudfoundry/app-autoscaler/blob/df1c9a05a00d927b69d2fbf885f32e0b4f228359/docs/Readme.md#schedules suggests not)

Also, I'm not really sure how we define non-overlapping schedules - perhaps if we are explicitly speccing we should test out on staging so we know what it's behaviour is - especially given https://github.com/cloudfoundry/app-autoscaler/blob/df1c9a05a00d927b69d2fbf885f32e0b4f228359/docs/policy.md#constraints

Reply:
I think if your shedules pick up from each other then the defaults should never come into play.
but I was unsure if they're using `>=` or `>`, so on the off chance there's a gap of a second I thought some defaults might be a good idea?

I'll do some research into the schedule rules next week and remove that if it's impossible it is ever called?